wgpu-core/src/device/global.rs

```diff
@@ -1897,6 +1900,24 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         };
 
         let id = fid.assign_error(desc.label.borrow_or_default(), &mut token);
+
+        // We also need to assign errors to the pipeline layout and the bind
```

Comment:
nit: "to the *implicit* pipeline layout" it's easy to forget while reading this code that implicit layouts exist.